k8s/src/main/scala/io/buoyant/k8s/Api.scala

```diff
@@ -0,0 +1,187 @@
+package io.buoyant.k8s
```

Comment:
good question: I was trying to have things match the URL structure. The third-party namespaces are outside the core of the v1 API (i.e. they start with `apis/owner/owner-specified-version` rather than `api/v1`), so I'm not sure how they map into the version of the API as a whole. How would we want to differentiate between the core v1 API and things like `apis/extensions/v1beta1`, etc.?